Update BBE description for path-parameters (#3783)
ballerina-platform/ballerina-distribution: examples/http-path-param/http_path_param.md

# REST service - Path parameter

> The `Path parameter` is a mandatory but variable part of a resource URL. `Path parameters` can be added to a resource method by specifying the parameter type and name in the resource method signature (eg: `albums/[string name]`). The `http` module supports `string`, `int`, `float`, `boolean`, and `decimal` types in path parameters. Use it when designing a REST API endpoint which should uniquely identify a resource or resources.

Reviewer: ```suggestion
The `path parameter` is a mandatory but variable part of a resource URL. `Path parameters` can be added to a resource method by specifying the parameter type and name in the resource path (eg: `albums/[string name]`). The `http` module supports `string`, `int`, `float`, `boolean`, and `decimal` types as path parameter types. Use it when designing REST API endpoints that require dynamic path segments.
```
